app/models/cart.rb

```diff
@@ -203,4 +203,9 @@ def total_price
       0.0
     end
   end
+
+  # may be replaced with paper-trail or similar at some point
```

Comment:
I'm not sure I follow. If we add paper-trail or something similar, we'd keep the `version` name, but back it with the paper-trail data (rather than updated_at data). Maybe `version` is a keyword with paper-trail, though, and the interface should be `version_name` or something?